### DISCUSSION TITLE
Logo in printable QR code

### DISCUSSION THREAD

**Discussion by iBobik** [btcpayserver/btcpayserver#3632] [Ideas / Feature Requests]
POS app offers static QR codes for print.

It would be nice to offer some more visual controlls for QR codes, like redundancy or logo.

I usually use https://www.qrcode-monkey.com to add this logo:
![bitcoin-lightning-qr-icon](https://user-images.githubusercontent.com/614232/163182565-991a2faa-b5f0-41d2-8b74-f7efe92516b9.png)

**Comment by dstrukt** [comment]
Would be a nice feature to add for some customization.

Noted!
